test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -288,9 +288,9 @@ def _set_file(self, hostname, fin, fout, append, vars):
             with open(fn, 'w') as fd:
                 for k, v in conf.items():
                     fd.write(str(k) + '=' + str(v) + '\n')
-            self.run_copy(hostname, fn, fout, sudo=True,
-                          preserve_permission=False, level=logging.DEBUG2)
-            self.rm(path=fn)
+            self.chown(path=fn, uid=0, gid=0, sudo=True)
```

Comment:
Why you reverted this back to original code (aka without preserve_permission) ?

Reply:
This is required as conf files are owned by root user if we don't preserve permission then the new file will get permission of 'fout' parameter .